.github/workflows/build.yml

```diff
@@ -0,0 +1,65 @@
+name: build
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: macos-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        api-level:
+          - 17
```

Comment:
It appears that `reactivecircus/android-emulator-runner@v2` supports down to API Level 15. 

I'd suggest we move to support API Level 16, and test for that here, as this was the first Jelly Bean release. API Level 17 seems a little odd as it was an update to Jelly Bean - unless we need something specifically in 17 onwards? 

Either way, this release should take us back to API Level 14 as it's a minor release then with 1.5.0 we can make the decision to move to 16+. Correct? Unless we're going to say from 1.4.0 we support API Level 16+ only (and we just update the docs to make that clear and also make it clear on 1.4.0 and 1.4.1 that an issue means API Level is 21+).

Reply:
Ah right, I actually forgot the reason why I've pushed that to API 17 (so many attempts) I think it was something related to failures not directly related to the code but some weirdness with the testing environment. I can give another try to lower API as meanwhile I've fixed various code issues and maybe they affected the testing at that age.

By the way, I'm not too keen in a perfect match between min tracker's supported version (14) and tested version (hopefully 16 - now 17) because we are doing already a giant step forward testing on three different platforms rather than a single one (24).

I would bump to API 16+ in the tracker 2.0 as it's technically a breaking change.
As long as we are doing minor changes in the tracker I think we can just keep compatibility with API 14 also if not tested. I don't think there are many apps below 16 and we haven't tested our tracker below 16 in our last releases.

Reply:
For future reference:

It seems that API 16 fails due to...
```
com.android.ddmlib.InstallException: Unknown failure: Error: Unknown option: -d
usage: pm list packages [-f] [-d] [-e] [-s] [-3] [-i] [-u] [FILTER]
```

Which is probably caused by annoying version incompatibility and other configuration problems. I don't think it's worth to spend more time on that now.
I add a note about the issue in the script - so I don't forget that again!